### ISSUE TITLE
Icons too small in iOS app

### ISSUE THREAD

**Issue by gurbyz** [gurbyz/power-wheel-card#50]
I have read the wiki about what to try first before creating a bug report.

**Which version of HA do you use?**
0.110.2
iOS app 2.0.0(69)

**Which version of the card do you use?**
0.1.1

**Describe the bug**
On a desktop browser icon size is correct.
On a mobile browser icon size is correct.
In the iOS app icon size of the home/solar/grid icons is too small.

**Configuration of the card**
Copy the complete configuration of the card, even if you think it isn't relevant.
```
  - type: "custom:power-wheel-card"
    solar_power_entity: sensor.solar_power
    grid_power_consumption_entity: sensor.grid_power_consumption
    grid_power_production_entity: sensor.grid_power_production
    solar_energy_entity: sensor.solar_energy_today
    grid_energy_consumption_entity: sensor.grid_energy_consumption_today
    grid_energy_production_entity: sensor.grid_energy_production_today
    energy_consumption_rate: 0.1883
```

**Expected behavior**
Bigger size icons in iOS app.

**Comment by gurbyz** [comment]
Because of recent changes in HA per 0.110.x I guess the iOS app might need a change for this too. Since the small window size of mobile is not the problem. (Icon size is correct on emulated mobile windows and on my mobile Chrome browser).

**Comment by gurbyz** [comment]
Got magically fixed in HA 0.110.3.
Maybe it was: Update frontend to 20200519.5 (@bramkragten - #36154) (frontend docs)
